src/main/java/org/reaktivity/nukleus/kafka/internal/function/StringIntKafkaErrorConsumer.java

```diff
@@ -15,8 +15,10 @@
  */
 package org.reaktivity.nukleus.kafka.internal.function;
 
+import org.reaktivity.nukleus.kafka.internal.stream.KafkaError;
+
 @FunctionalInterface
-public interface StringIntShortConsumer
+public interface StringIntKafkaErrorConsumer
```

Comment:
We probably want _either_ the types in the functional interface name, _or_ call it `KafkaErrorConsumer`, but not _both_ approaches at the same time.

Reply:
Renamed to KafkaErrorConsumer